### DISCUSSION TITLE
Seeking tips for <leader> shortcuts

### DISCUSSION THREAD

**Discussion by lcfd** [numToStr/Comment.nvim#268] [Q&A] [answered]
I would like to set up shortcuts that allow me to comment using the `<leader>` key.
At the moment I only managed to set that with `<leader>1` comment the current line. 
If I press `6<leader>1` it comments the next 6 lines.

Is it possible to make `<leader>1` work in visual mode on multiple lines?
How do you use it?

Thank you! 😊

**Comment by numToStr** [comment, marked as answer]
> Is it possible to make <leader>1 work in visual mode on multiple lines?

If you configured it via `opleader.line` then it should also work in visual mode, including normal mode.

But, if you want `<leader>1` to only work in visual mode (no normal mode) then it's not possible to configure via `setup()`.

Beside that, you could use `:h comment.plugmaps` to define your own custom keybindings.

**Reply by lcfd** [reply]
Hi, thank you for the answer.

It seems working! I still have to understand how block-wise things work but it's more than fine.
